### DISCUSSION TITLE
Maximum number of clusters

### DISCUSSION THREAD

**Discussion by mongotae** [minio/minio#15265] [Q&A] [answered]
Can you share whether the maximum number of clusters or nodes are 32 or more than it?
If it is unlimited, is there any references to use it in production level?

I designed that there are multiple clusters (scaling out infinitely) with 32 servers, including 16 drives per server.
I want to figure out whether it makes sense in the production level.

**Comment by klauspost** [comment, marked as answer]
@mongotae There are no limits to MinIO server count.

To discuss your production setup, please write `sales@min.io`, and we can arrange for a thorough review before you go into production.
